main.go

```diff
@@ -241,11 +267,21 @@ func searchChart(r []*search.Result, name string, chartVersion string, devel boo
 
 	if !found {
 		debug("Could not find any Repo which contains %s", name)
-		return nil, errors.New(fmt.Sprintf("Could not find any Repo which contains %s", name))
+		return nil, false, errors.New(fmt.Sprintf("Could not find any Repo which contains %s", name))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [should replace errors.New(fmt.Sprintf(...)) with fmt.Errorf(...)](https://app.codacy.com/manual/l0nax/helm-whatup/pullRequest?prid=4860442)